src/app/App.tsx

```diff
@@ -15,15 +18,17 @@ const App: React.FC<RouteComponentProps> = ({
   const orderConfirmationPage = isPath(pathname, orderConfirmationUrl);
 
   return (
-    <>
-      <MetaConsumer />
-      <header>
-        <MainMenu />
-      </header>
-      <Routes />
-      {!orderConfirmationPage && <Footer />}
-      <OverlayManager />
-    </>
+    <ThemeProvider theme={theme}>
+      <>
```

Comment:
I guess we dont need empty element anymore?

Reply:
ThemeProvider expects a single child and will otherwise throw a TS error

Reply:
👍